wordpress-mobile/gutenberg-mobile
# Blocks no longer merge when pressing backspace with a non-zero-length selection.

### Description:

Title has it.

Fixes #325

### Related PRs:

There's a `react-native-aztec` change that should be reviewed as part of this PR [here](https://github.com/wordpress-mobile/react-native-aztec/pull/95).

### Testing:

#### Test 1:

1. Launch the editor demo.
2. Make sure you have two consecutive blocks that can be merged (like two paragraph blocks).
3. Select the first word from the second block.
4. Press backspace.

Expected behavior: the blocks should not merge.  The word should be deleted.

#### Test 2:

1. Launch the editor demo.
2. Make sure you have two consecutive blocks that can be merged (like two paragraph blocks).
3. Place the caret at the beginning of the second block, before any text.
4. Press backspace.

Expected behavior: the blocks should merge.
